khmer/khmer_args.py

```diff
@@ -389,18 +393,22 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0, _sysmem=None):
```

Comment:
The downside to having the argument there is that one day a user will try and use it. I don't think it will cause technical problems. The pythonista in me thinks it is ugly, in the end it is like all code smells...